docs/docs/setup/digitalocean.md

```diff
@@ -1,113 +1,74 @@
-:::
-
-## Deploying ToolJet Database
+**3. Create a Firewall for the **Droplets** to allow required ports.**
+   
+   protocol | port     | allowed_cidr|
+   :---| :----------  | :---------- |
+   tcp | 22           | your IP     |
+   tcp | 80           | 0.0.0.0/0   |
+   tcp | 443          | 0.0.0.0/0   |
 
-To use the ToolJet Database, you need to set up and deploy a PostgREST server, which facilitates querying the database. Detailed setup instructions are available [here](/docs/tooljet-db/tooljet-database).
+**4. Connect to the **Droplets** via SSH.**
+ 
+**5. Install Docker and Docker Compose using the following commands:**
 
-Starting with ToolJet 3.0, deploying the ToolJet Database is mandatory to avoid migration issues. Refer to the documentation below for details on the new major version, including breaking changes and required adjustments for your applications.
+```bash
+apt update && apt upgrade -y
+apt install -y docker.io
+```
 
-- [ToolJet 3.0 Migration Guide for Self-Hosted Versions](./upgrade-to-v3.md)
+Enable and start Docker:
 
-Follow the steps below to deploy ToolJet Database on DigitalOcean:
+```bash
+systemctl enable docker 
+systemctl start docker
+```
 
-1. If you are using dev database within ToolJet deployment, upgrade it to managed database. You could also add a separate database, if you intent use a different database, please refer the [environment variables](/docs/setup/env-vars#enable-tooljet-database-required) for additional env variables. 
+Install Docker Compose:
 
-2. Create a new app for PostgREST server. You can opt for docker hub to deploy PostgREST image of version `10.1.x`.
+```bash
+apt install -y curl 
+curl -SL https://github.com/docker/compose/releases/latest/download/docker-compose-linux-x86_64 -o /usr/local/bin/docker-compose 
+chmod +x /usr/local/bin/docker-compose
+```
 
-  <img className="screenshot-full" src="/img/setup/digitalocean/postgrest-build.png" alt="ToolJet - Deploy on DigitalOcean - PostgREST resource" />
+Verify installation:
 
-3. Update the [environment variables](/docs/setup/env-vars#postgrest-server-required) for PostgREST and expose the HTTP port `3000`.
+```bash
+docker --version 
+docker-compose --version
+```
 
-  <img className="screenshot-full" src="/img/setup/digitalocean/postgrest-env.png" alt="ToolJet - Deploy on DigitalOcean - PostgREST environment variables" />
+**6. Update the `TOOLJET_HOST` in the `.env` file:**
 
-4. Add your newly created PostgREST app to the trusted sources of your managed or separate database.
+`TOOLJET_HOST=http://<public_ip>:80`
 
-5. Update your existing ToolJet application deployment with [environment variables](/docs/setup/env-vars#enable-tooljet-database-required) required for PostgREST. 
+**7. Use the [Docker Documentation](https://docs.tooljet.com/docs/setup/docker) to deploy ToolJet.**
```

Comment:
For 3.1.0 version doc. This url needs to be - https://docs.tooljet.com/docs/beta/setup/docker. @karan-rathod-316